journal-letters/modernized/13-to-barbados.adoc

```diff
@@ -166,30 +164,30 @@ And if he will not hear, nor be gained, then says Christ,
 
 But he gives no commission to Christians to persecute, to put in prison,
 to take away goods, to pull down their houses, to put their feet in the stocks,
-to root them out of the world root and branch; no,
+to root them out of the world, root and branch; no,
 nor to wish it so to be done unto them.
-But if any do, he will reprove such, as he did the disciples, when he told them.
-They knew not what spirit they were of, when they desired fire to come from heaven;
+But if any do, he will reprove such, as he did the disciples, when he told them,
+"`They knew not what spirit they were of,`" when they desired fire to come from heaven;
```

Comment:
This one, because it's not a direct quotation of scripture (he changed the _person_ from first to third) I think we should either remove the quotes, or only do italics.  I think I slightly lean towards the former because of the informalness of the whole reference.